db/db.js

```diff
@@ -2,6 +2,15 @@
  * This file is used for connecting to MongoDB.
  */
 
+// module.exports = function () {
```

Comment:
Let remove this if it is not needed.

Reply:
have kept the code commented, because some of us couldn't connect to db locally, so we used the commented code to connect locally.

Reply:
Okay